QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
-QUEST_LV_0100_20150317_004213	Find the clues of the disappearance by defeating Sauga
-QUEST_LV_0100_20150317_004214	The mercenary Glen seems to care about the disapperance incident that occured recently. Check if there are some clues in the items that you will obtain after defeating Sauga.
-QUEST_LV_0100_20150317_004215	The items that came out from Sauga's pocket seem useless. Talk to Glen again.
-QUEST_LV_0100_20150317_004216	Fortunately, the items don't belong to Irene. Talk to Glen again.
-QUEST_LV_0100_20150317_004217	Defeat the monsters on the way to Rodoma Archeological Site.
-QUEST_LV_0100_20150317_004218	Glen asked you to defeat the monsters that turned out violent since no one knows what would happen in the future. Defeat the monsters that are on the way of Rodoma 1st Archeological Site.
-QUEST_LV_0100_20150317_004219	It will be safe since you defeated all the monsters that turned out violent. Talk to Glen again.
-QUEST_LV_0100_20150317_004220	Defeat Ticen and Tucen
-QUEST_LV_0100_20150317_004221	The assitant Irene seems alright. Talk to Irene again.
-QUEST_LV_0100_20150317_004222	Protect Irene from the monsters
-QUEST_LV_0100_20150317_004223	Irene told you that she got into the accident while following some man. Who the hell did this? Anyways, protect Irene since the monsters are coming.
-QUEST_LV_0100_20150317_004224	Irene seems alright. Talk to Irene again.
-QUEST_LV_0100_20150317_004225	Talk to Irene. She is trembling.
-QUEST_LV_0100_20150317_004226	Go back to Dezik again
-QUEST_LV_0100_20150317_004227	Irene returned to Dezik. Check whether Irene returned to Dezik safely.
-QUEST_LV_0100_20150317_004228	Dezik fell down after she got attacks from Scorpio! Talk to Irene again.
-QUEST_LV_0100_20150317_004229	Defeat Scorpio
-QUEST_LV_0100_20150317_004230	Talk to the New Researcher of the Royal Mausoleum
+QUEST_LV_0100_20150317_004201	$Archeologist Dezic is desperately looking for someone. Try talking to him.
+QUEST_LV_0100_20150317_004202	$Talk to Archeologist Dezic.
+QUEST_LV_0100_20150317_004203	$Archeologist Dezic informed you that Irene is imprisoned with barrier devices in Rodoma 1st Ruin. You must hurry and find her.
+QUEST_LV_0100_20150317_004204	$Talk to Assistant Irene
+QUEST_LV_0100_20150317_004205	$You found Assistant Irene. Get to her cautiously to avoid the monsters harming her.
+QUEST_LV_0100_20150317_004206	$Talk to Irene
+QUEST_LV_0100_20150317_004207	$Irene can't move since she is binded to some barrier devices. Talk to Irene.
+QUEST_LV_0100_20150317_004208	$Remove the barrier devices from Irene
+QUEST_LV_0100_20150317_004209	$Remove all the devices from Irene. The three devices are at the west, east, and north side of the site.
+QUEST_LV_0100_20150317_004210	$Removed all the devices. Check if Irene is okay.
+QUEST_LV_0100_20150317_004211	$Talk to Mercenary Glen
+QUEST_LV_0100_20150317_004212	$Glen needs your help.
+QUEST_LV_0100_20150317_004213	$Defeat Saugas and find clues about the missing researchers
```

Comment:
If you want to stay true to the original, it should be along the lines of "Defeat Saugas to find a clue about the disappearance." - Because it doesn't say researchers in the original.